builtin/aws/ecs/platform.go

```diff
@@ -3303,6 +3320,25 @@ deploy {
 		"the instruction set CPU architecture that the Amazon ECS supports. Valid values are: \"x86_64\", \"arm64\"",
 	)
 
+	doc.SetField(
+		"target_group_protocol",
+		"The protocol to use for routing traffic to the targets.",
+		docs.Default("HTTP"),
+		docs.Summary("The protocol to use for routing traffic to the targets. "+
```

Comment:
I pulled the descriptions for these two fields from the [AWS docs](https://docs.aws.amazon.com/elasticloadbalancing/latest/APIReference/API_CreateTargetGroup.html) - I wasn't sure if that was kosher for our docs, so if it's not, happy to change!